js/ui/scheduler/workspaces/ui.scheduler.work_space.js

```diff
@@ -929,10 +929,9 @@ class SchedulerWorkSpace extends WidgetObserver {
     _getRowCount() { return noop(); }
 
     _getRowCountWithAllDayRows() {
-        const allDayRowsCount = this.option('showAllDayPanel')
-            ? this._getGroupCount() : 0;
+        const allDayRowCounted = this.option('showAllDayPanel') ? 1 : 0;
```

Comment:
`const allDayRowCounted = +this.option('showAllDayPanel');` ? :)

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Unary_plus

Reply:
To me this seems less clear. But if you insist, I will do it)

Reply:
`+this.option('showAllDayPanel')` is not clear to me too.
I think we should rename: `allDayRowCounted` -> `allDayRowCount`